.github/ISSUE_TEMPLATE/generic-docs.md

```diff
@@ -8,7 +8,7 @@ labels: docs, status/triage
 **Description**:
 
 <!--
-Please describe what you want to change for the documentation.
+Please describe what documentation should be changed.
 -->
 
 **Rationale**:
```

Comment:
You have used `# HEADLINE` semantics in other templates, I think we should either use the current version (`**HEADLINE**:`) or the new version (`# HEADLINE`) in all templates.